client.go

```diff
@@ -10,6 +10,7 @@ type Client interface {
 	Create(*kinesis.CreateStreamInput) (bool, error)
 	Status(*kinesis.DescribeStreamInput) string
 	Tag(*kinesis.AddTagsToStreamInput) error
+	Kinesis() *kinesis.Kinesis
```

Comment:
Might be better to just add a PutRecords method here and make newFlusher take a `Client` interface.